Toolset/libraries/revbackscriptlibrary.livecodescript

```diff
@@ -2572,7 +2581,7 @@ function revDatabaseDriverBuildList
    local tList
    put empty into tList
    // AL-2015-02-17: [[ SB Inclusions ]] Ensure mysql module name is standard (was dbmysql)
-   repeat for each item tExternal in revIDEDatabaseDriversMapping()
+   repeat for each item tExternal in "MySQL mysql,ODBC odbc,PostgreSQL postgresql,SqLite sqlite,Oracle oracle"
```

Comment:
@montegoulding I think these changes are unintentional -- could you please remove them from the patch?